qutebrowser/__init__.py

```diff
@@ -26,8 +26,10 @@
 __license__ = "GPL"
 __maintainer__ = __author__
 __email__ = "mail@qutebrowser.org"
-__version_info__ = (1, 5, 0)
-__version__ = '.'.join(str(e) for e in __version_info__)
+__version__ = "1.5.0"
+__version_info__ = [int(part) for part in __version__.split('.')]
 __description__ = "A keyboard-driven, vim-like browser based on PyQt5."
 
 basedir = os.path.dirname(os.path.realpath(__file__))
+appdata_path = os.path.join(os.path.dirname(basedir), "misc",
```

Comment:
I'm a bit confused about the `os.path.dirname(basedir)` here - is this just to get to the parent directory? Either way, I think I'd prefer just having this in `update_version.py` as it's not needed in qutebrowser itself.